src/Storage/appsettings.json

```
@@ -24,6 +24,7 @@
     "RuntimeCookieName": "AltinnStudioRuntime",
     "BridgeApiAuthorizationEndpoint": "http://localhost:5055/sblbridge/authorization/api/",
     "OndemandEndpoint": "http://localhost:5050/storage/api/v1/",
```

5010 er vel storage porten ? 

```suggestion
    "OndemandEndpoint": "http://localhost:5010/storage/api/v1/",
```